content/300-guides/050-database/850-planetscale.mdx

```diff
@@ -225,6 +225,74 @@ model User {
 
 For a more detailed example, see the [Getting Started guide for PlanetScale](/getting-started/setup-prisma/add-to-existing-project/relational-databases/introspection-typescript-planetscale).
 
+## How to use the PlanetScale serverless driver with Prisma (Preview)
```

Comment:
Should we have a link to npm `@prisma/adapter-planetscale` in here somewhere?

Reply:
Yeah, sure!